shared-bindings/busio/SPI.c

```diff
@@ -73,7 +73,9 @@
 //|         clock: microcontroller.Pin,
 //|         MOSI: Optional[microcontroller.Pin] = None,
 //|         MISO: Optional[microcontroller.Pin] = None,
+//|         SS: Optional[microcontroller.Pin] = None,
 //|         half_duplex: bool = False,
+//|         slave_mode: bool = False,
```

Comment:
Please make a new module for this such as `spitarget` (to match `i2ctarget`). That will greatly reduce the impact of this change because existing constructors won't need to be changed.

We also prefer terminology that isn't historically loaded: https://docs.circuitpython.org/en/latest/docs/design_guide.html#terminology So, please don't use the terms `master` or `slave` in the interfaces. It is ok to document that they were previously called this.

Reply:
I've moved the secondary-mode behavior to a new spitarget module and fixed the naming, see https://github.com/Bruin-Spacecraft-Group/bruinspace-circuitpython/tree/main/shared-bindings/spitarget